addons/nodot/interaction/Teleport3D.gd

```diff
@@ -0,0 +1,82 @@
+##  Teleport3D is an Area3D which, when touched,
```

Comment:
These three can squeeze onto one line. It's so that the documentation generator can read it correctly.